src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -292,36 +305,94 @@
 (def required-cli-options
   [:config])
 
+(defn activate-logging!
+  [options]
+  (-> (:config options)
+      (ks/inis-to-map)
```

Comment:
No need for these parens, but it looks like this is really just moved from down below